linera-core/src/worker.rs

```diff
@@ -462,24 +462,9 @@ impl<StorageClient> WorkerState<StorageClient>
 where
     StorageClient: Storage + Clone + Send + Sync + 'static,
 {
-    // NOTE: This only works for non-sharded workers!
-    #[instrument(level = "trace", skip(self, certificate, blobs))]
-    #[cfg(with_testing)]
-    pub async fn fully_handle_certificate<T>(
-        &self,
-        certificate: GenericCertificate<T>,
-        blobs: Vec<Blob>,
-    ) -> Result<ChainInfoResponse, WorkerError>
-    where
-        T: ProcessableCertificate,
-    {
-        self.fully_handle_certificate_with_notifications(certificate, blobs, &())
-            .await
-    }
-
     #[instrument(level = "trace", skip(self, certificate, notifier))]
     #[inline]
-    pub(crate) async fn fully_handle_certificate_with_notifications<T>(
```

Comment:
(We could just call this one `fully_handle_certificate` now. I'd say the `notifier` argument itself documents that this will create notifications, too.)

Reply:
Will do that in a separate PR to not wait for the CI again